distributed/scheduler.py

```diff
@@ -7675,7 +7675,7 @@ async def _track_retire_worker(
                 f"Could not retire worker {ws.address!r}: unique data could not be "
                 f"moved to any other worker ({stimulus_id=!r})"
             )
-            return ws.address, "no-recipients"
+            return ws.address, "no-recipients", ws.identity()
```

Comment:
I could be missing something, but it looks like we're adding back the verbose thing removed in https://github.com/dask/distributed/pull/8935 now. Just wanted to make sure that's intentional